tests_new/integration_tests/modules/s3_datasets/test_s3_tables_filters.py

```diff
@@ -0,0 +1,149 @@
+import logging
+
+from assertpy import assert_that
+import pytest
+
+from integration_tests.modules.s3_datasets.queries import (
+    create_table_data_filter,
+    delete_table_data_filter,
+    list_table_data_filters,
+)
+from integration_tests.errors import GqlError
+
+log = logging.getLogger(__name__)
+
+COL_INPUT = {
+    'filterName': 'columnfilter',
+    'description': 'test column',
+    'filterType': 'COLUMN',
+    'includedCols': ['col_0'],
+}
+ROW_INPUT = {
+    'filterName': 'rowfilter',
+    'description': 'test row',
+    'filterType': 'ROW',
+    'rowExpression': 'col_1 LIKE "%value%" AND col_0 IS NOT NULL',
+}
+
+
```

Comment:
nit: Although defining tests that depend on each other is not wrong, it is best practice to make tests not depend on each other if possible (for profiling jobs I ended up not doing it because there is no way of deleting a run). 

In this case it would mean having a data filter fixture that is checked in the `create-*` test, imported in the `list-*` test and `delete-*` test would create their own filter. 

The second advantage of creating the filter as a fixture is that we can add it to the globconftest and use it in the shares tests.

Reply:
If going this route and use the parametrized fixtures without dependencies, check the changes in #1391 that avoid duplicating lines of code with `@pytest.mark.parametrize(*TABLES_FIXTURES_PARAMS)`